transport_nantes/photo/views.py

```diff
@@ -49,3 +53,14 @@ class Confirmation(TemplateView):
     Confirmation page after successful submit of a PhotoEntry
     """
     template_name = 'photo/confirmation.html'
+
+    def get(self, request, *args, **kwargs):
+        logger.info(f"{request.user.email} landed on confirmation page.")
+        return super().get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        last_submitted_photo = \
+            PhotoEntry.objects.filter(user=self.request.user).last()
```

Comment:
Doesn't this create a race condition if the user is submitting in two windows simultaneously?

Can we put the data in the context when we process the upload POST and so have it available here in the context?